types/glob/index.d.ts

```diff
@@ -34,7 +38,7 @@ declare namespace G {
         silent?: boolean;
         strict?: boolean;
         cache?: { [path: string]: boolean | 'DIR' | 'FILE' | ReadonlyArray<string> };
-        statCache?: { [path: string]: false | { isDirectory(): boolean} | undefined };
+        statCache?: { [path: string]: false | Stats | undefined };
```

Comment:
I intentionally changed it to this type. I use this option as input to mock certain files. Since the implementation only uses `isDirectory` I don't need to provide a complete implementation of `Stats`.

I guess your use case is the exact opposite and you want to reuse the statCache in your app to avoid calling `fs.stat` again?

Reply:
> I guess your use case is the exact opposite and you want to reuse the statCache in your app to avoid calling fs.stat again?

Correct 👍